### ISSUE TITLE
Problems with saving as HTML 

### ISSUE THREAD

**Issue by abhitrueprogrammer** [al-caughey/Google-Meet-Attendance#83]
When Clicking save as HTML button in the extension nothing is actually done and no file is saved onto my PC, I was able to replicate this issue both in firefox and chrome. Using windows 10 2004 as the OS.

**Comment by AlCaughey** [comment]
Yes, there's an error. I'm trying to get an update out but my day job keeps
interfering
